src/els_completion_provider.erl

```diff
@@ -258,6 +260,7 @@ keywords() ->
 %% Filter by prefix
 %%==============================================================================
 
+%% TODO: Implement as select
```

Comment:
Most of the filtering/selecting in the providers can be implemented as Mnesia queries.